src/test/java/org/cactoos/collection/ImmutableTest.java

```diff
@@ -48,8 +47,8 @@ void size() {
             "size() must be equals to original",
             new Immutable<>(
                 new ListOf<>(1, 2)
-            ),
-            new HasSize(2)
+            ).size(),
```

Comment:
@andreoss Please, use `LengthOf` instead of calling `#size()`.

Reply:
@fabriciofx `#size()` is used explicitly for coverage, `LengthOf`  has the same problem as `HasSize` (i.e counting elements via `#iterator()`)